blocked-edges/4.12.1-OVNCrashOnDualStackMigrated.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.12.1
+from: .*
+url: https://issues.redhat.com/browse/SDN-4013
+name: OVNCrashOnDualStackMigrated
```

Comment:
Can we change it to `OVNCrashOnMigratedDualStack` ?

Reply:
Sure